src/rpc/net.cpp

```diff
@@ -1103,10 +1103,12 @@ UniValue AddrmanEntryToJSON(const AddrInfo& info)
     ret.pushKV("network", GetNetworkName(info.GetNetClass()));
     ret.pushKV("source", info.source.ToStringAddr());
     ret.pushKV("source_network", GetNetworkName(info.source.GetNetClass()));
+    ret.pushKV("mapped_as", connman.GetMappedAS(info));
+    ret.pushKV("source_mapped_as", connman.GetMappedAS(info.source));
```

Comment:
nit: Looking at my node's `getrawaddrman` output, it felt odd to have `mapped_as` appear after `source`. IMO, the fields become more intuitively understandable when put right after or at least close to the address they refer to.

Reply:
I agree, will address it.